3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/EnrichmentConfigsSpec.scala

```diff
@@ -360,4 +360,87 @@ class EnrichmentConfigsSpec extends Specification with ValidationMatchers {
       }
     }
   }
+
+  "Parsing an iab_spiders_and_robots_enrichment JSON" should {
+    "successfully construct an IabEnrichment case class if JSON is valid" in {
+
+      val iabJson = parse("""{
+        "enabled": true,
+        "parameters": {
+          "ipFile": {
+            "database": "ip_exclude_current_cidr.txt",
+            "uri": "https://example.com"
```

Comment:
let's add a test incorporating my remark above

Reply:
that was meant to add a / at the end of `uri` to check uri resolving works properly